### PR TITLE
US21

### PR DESCRIPTION
**Pull request by dtsehl**
Implemented user story 21 - user can edit their password

### CODE REVIEW

File: app/controllers/users_controller.rb [hunk 1]
```diff
@@ -36,6 +36,18 @@ def update
     end
   end
 
+  def edit_password
```
Comment (dtsehl) [inline app/controllers/users_controller.rb:39]: Technically the #edit_password and #update_password methods are not ReSTful routes, but these wound up working.

File: app/controllers/users_controller.rb [hunk 2]
```diff
@@ -45,4 +57,8 @@ def user_params
   def edit_user_params
     params.permit(:name, :address, :city, :state, :zip, :email)
   end
+
+  def password_params
```
Comment (dtsehl) [inline app/controllers/users_controller.rb:61]: Needed another params to allow just the password to be updated (as password is updated on a separate page from other user info.

**Review by Alex-Desjardins — APPROVED** [review]
Great work!

File: spec/features/users/edit_spec.rb [hunk 3]
```diff
@@ -36,5 +36,26 @@
 
       expect(page).to have_content("Joe")
     end
+    it "allows me to click a link to edit my password" do
```
Comment (michaeljevans) [inline spec/features/users/edit_spec.rb:39]: this test looks good

File: app/controllers/users_controller.rb (hunk 1 above)
Comment (michaeljevans) [reply app/controllers/users_controller.rb:39]: if it works, it works! maybe we can ask Cory about this at the next project check-in

**Pull request merged**